File: 976. Largest Perimeter Triangle.java
class Solution {
    public int largestPerimeter(int[] nums) {
        int n = nums.length;
        int i = n-3;
        int j = n-1;
        Arrays.sort(nums);
       while(i>=0){
          
         if(nums[j]<nums[i]+nums[i+1]){
           return nums[j]+nums[i]+nums[i+1];
         }
         
          i--;
          j--;
       }
      return 0;
    }
}

File: October leetcode challenge/Day12/976. Largest Perimeter Triangle.cpp
class Solution {
public:
    int largestPerimeter(vector<int>& nums) {
      sort(nums.begin(),nums.end());
      for(int i=nums.size()-3;i>=0;i--)
      {
          if(nums[i]+nums[i+1]>nums[i+2])
          {
              return nums[i]+nums[i+1]+nums[i+2];
          }
      }
        return 0;
    }
};
